AzureCalling/Controllers/CallViewController.swift

```diff
@@ -35,7 +40,7 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
     @IBOutlet weak var toggleVideoButton: UIButton!
     @IBOutlet weak var toggleMuteButton: UIButton!
     @IBOutlet weak var infoHeaderView: InfoHeaderView!
-    @IBOutlet weak var messageBannerView: MessageBannerStackView!
+    @IBOutlet weak var messageBannerStackView: MessageBannerStackView!
```

Comment:
Since we are using the name NoticeBannerType, I think we can rename this to `noticeBannerStackView` and `NoticeBannerStackView` to be consistent.